elytron/src/main/java/org/wildfly/extension/elytron/ModifiableRealmDecorator.java

```diff
@@ -418,6 +418,7 @@ protected void executeRuntimeStep(final OperationContext context, final ModelNod
                             DigestPassword.ALGORITHM_DIGEST_MD5,
                             DigestPassword.ALGORITHM_DIGEST_SHA,
                             DigestPassword.ALGORITHM_DIGEST_SHA_256,
+                            DigestPassword.ALGORITHM_DIGEST_SHA_384,
```

Comment:
This requires an addition to ElytronSubsystemTransformers, tranforming to ELYTRON_1_2_0.

Reply:
@bstansberry As we have commented by email, I have verified that set-password operation is not registered under /profile=XXXX/ path because it's a runtime op, so it looks like unnecessary to register a transform here against to 1.2.0, Are you agree?

Reply:
@yersan Yes, I agree. This AttributeDefinition is only used by SetPasswordHandler, and ModifiableRealmDecorator.registerOperations only registers that hander on a server or an HC subsystem, not on a domain-wide /profile=* resource where a transformer is required.

Sorry for steering you in the wrong direction!